pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -456,8 +448,6 @@ func (r *ReconcileHyperConverged) validateNamespace(req *common.HcoRequest) (boo
 
 func (r *ReconcileHyperConverged) setInitialConditions(req *common.HcoRequest) {
 	req.Instance.Status.UpdateVersion(hcoVersionName, r.ownVersion)
-	req.Instance.Spec.Version = r.ownVersion
-	req.Dirty = true
```

Comment:
why are removing setting the dirty flag to `true` if you update the CR status?

Reply:
`Dirty` is for the spec/metadata. `StatusDirty` is dedicated to status changes.